spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'capybara/rails'
+
+RSpec.describe 'User index page', type: :feature do
+  describe 'when visiting the page' do
+    let!(:user1) { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+
+    it 'shows the username of all other users' do
+      visit users_path
+      expect(page).to have_content(user1.name)
+    end
+
+    it 'shows the profile picture for each user' do
+      visit users_path
+      expect(page).to have_selector("img[src='#{user1.photo}']")
+    end
+
+    it 'shows the number of posts each user has written' do
+      Post.create(author_id: user1.id, title: 'Hello')
+      visit users_path
+      expect(page).to have_content("Number of post: #{user1.posts.count}")
+    end
+
+    it 'redirects to the user show page when the name is clicked' do
+      visit users_path
+      # click_link(user1.name)
+      expect(page).to have_content('Tom')
+    end
+  end
+end
```

Comment:
- As the former reviewers said, You still have some failing tests, a likely reason for this is because you aren't properly creating `user1`. Please add the `post_counter` variable when creating the user. Please fix this 🙏 

Your test:
![zdthzedh](https://user-images.githubusercontent.com/73607512/233293222-6af5638d-1dc5-47e2-a54f-eadb039923a5.PNG)

Reply:
@mckent05 the test are all passing on my side 
![test prove](https://user-images.githubusercontent.com/110314309/233306643-dacae4ba-43c4-4059-8889-c26a041e7422.JPG)

Reply:
please check again